### PR TITLE
Implement Inverse CDF dispatch functions

### PR DESCRIPTION
**Pull request by ricardoV94**

### CODE REVIEW

**Comment by ricardoV94** [comment]
Latest Aesara release is causing `tests/test_scan.py::test_initial_values` to fail:
https://github.com/aesara-devs/aeppl/runs/6918324742?check_suite_focus=true

**Review by brandonwillard — COMMENTED** [review]
After creating an issue for that unrelated, failing test, we can mark the test as `xfail` in this PR.

File: aeppl/logprob.py [hunk 1]
```diff
@@ -101,6 +109,22 @@ def _logcdf(
...
+    op: Op,
+    value: TensorVariable,
+    *inputs: TensorVariable,
+    **kwargs,
+):
+    """Create a graph for the icdf of a ``RandomVariable``.
+
+    This function dispatches on the type of ``op``, which should be a subclass
+    of ``RandomVariable``.  If you want to implement new icdf graphs
+    for a ``RandomVariable``, register a new function on this dispatcher.
```
Comment (brandonwillard) [inline aeppl/logprob.py:123]:
```suggestion
    """Create a graph for the inverse CDF of a `RandomVariable`.

    This function dispatches on the type of `op`, which should be a subclass
    of `RandomVariable`.
```

File: aeppl/logprob.py [hunk 2]
```diff
@@ -68,6 +68,14 @@ def logcdf(rv_var, rv_value, **kwargs):
     return logcdf
 
 
+def icdf(rv, value, **kwargs):
+    """Create a graph for the icdf of a ``RandomVariable``."""
```
Comment (brandonwillard) [inline aeppl/logprob.py:72]:
```suggestion
    """Create a graph for the inverse CDF of a `RandomVariable`."""
```

**Review by brandonwillard — COMMENTED** [review]
Just some minor updates; otherwise, this looks good.

**Review by brandonwillard — APPROVED** [review]

**Pull request merged**